opennms-doc/guide-admin/src/asciidoc/text/meta-data.adoc

```diff
@@ -1,29 +1,45 @@
-{opennms-product-name} supports the assignment of arbitrary meta-data to nodes, interfaces and services.
 
-The meta-data is a simple triple of strings containing a context, a key and the associated value.
-There can be an arbitrary number of meta-data elements assigned to each node, each interface and each service.
-The only restriction is, that the tuple of context and key must be unique in the element to which they are associated.
+// Allow GitHub image rendering
+:imagesdir: ../images
 
-The meta-data is associated to nodes, interfaces and services during link:#ga-provisioning-meta-data[provisioning].
+{opennms-product-name} supports the assignment of arbitrary metadata to nodes, interfaces and services. 
+You can then use this metadata to dynamically configure service monitoring, performance data collection, service detection, and expression-based thresholds. 
 
-To view the meta-data currently assigned to nodes, interfaces and services, there according details page can be used.
-These pages show the meta-data as it is currently assigned in the database.
+The metadata is a simple triad of strings containing a context, a key and the associated value.
+Each node, each interface and each service can have an arbitrary number of metadata elements assigned to it.
+The only restriction is that each context/key/value must be unique in the entity with which it is associated.
```

Comment:
This is somewhat misleading. This sounds like a user is able to define contextA/keyX/Value1 and contextA/keyX/Value2 at the same time, but a new value for a contextA/keyX will overwrite the old value. Of course key/context/value is unique, but because the tuple contextA/keyX is already unique itself.